src/IsoCodes/PhoneNumber.php

```diff
@@ -25,7 +26,11 @@ public static function validate($phoneNumber, $country)
         }
         $country = strtoupper($country);
         $phoneUtil = PhoneNumberUtil::getInstance();
-        $numberProto = $phoneUtil->parse($phoneNumber, $country);
+        try {
+            $numberProto = $phoneUtil->parse($phoneNumber, $country);
+        } catch (NumberParseException $e) {
```

Comment:
:-1: for handle `\Exception`. If the trowed exception is related to another issue, it will be obfuscated by out catch and the debug will be harder.

It will be even harder with PHP7 because many more error will be trowed as exception.

From doc: https://github.com/giggsey/libphonenumber-for-php/blob/master/docs/PhoneNumberUtil.md

> A NumberParseException will be thrown if it is unable to obtain a viable number. For example, if the number is too short/long, or the region is invalid. This does not tell you whether the number is valid or not. In order to determine whether the number is valid, it needs to be checked in the validation functions.

So this is the only exception to handle. If this is moving, I'll be glad to provide you another PR. ;-)